File: jcarrollonlinev4.backend/Models/MicroPosts/MicroPostFeedItemModel.cs
using jcarrollonlinev4.backend.Models.Users;

namespace jcarrollonlinev4.backend.Models.Micropost
{
    public class MicropostFeedItemModel : MicropostModelBase
    {
        public MicropostFeedItemModel()
        {
            Author = new ApplicationUserModel();
        }
        public ApplicationUserModel? Author { get; set; }
        public string? Content { get; set; }
        public DateTime CreatedAt { get; set; }
        public string? TimeAgo { get; set; }
    }
}

File: jcarrollonlinev4.frontend/src/models/MicroPosts/MicroPostFeedItemModel.ts
import { ApplicationUserModel } from "../Users/ApplicationUserModel";
import { MicropostModelBase } from "./MicroPostModelBase";

export interface MicropostFeedItemModel extends MicropostModelBase {
  Author?: ApplicationUserModel;
  Content?: string;
  CreatedAt: Date;
  TimeAgo?: string;
}
